packages/fxa-settings/src/constants/index.tsx

```diff
@@ -5,5 +5,3 @@
 export const ROOTPATH = '/';
 export const HomePath = '/settings';
 export const DeleteAccountPath = '/settings/delete_account';
-export const VPNLink = 'https://vpn.mozilla.org/';
-export const MonitorLink = 'https://monitor.firefox.com/';
```

Comment:
I think we actually use these in `ConnectedServices` as well as our Bento menu without referencing these consts. Could be worth leaving these, and changing those references to this const.

Reply:
```sh
git grep -nE "(monitor.firefox.com|vpn.mozilla.org)" | cat

packages/functional-tests/tests/signin/relyingParties.spec.ts:100:    await page.goto('https://monitor.firefox.com');
packages/functional-tests/tests/signin/relyingParties.spec.ts:106:    expect(page.url()).toMatch('https://monitor.firefox.com/user/dashboard');

packages/fxa-auth-server/test/local/payments/fixtures/stripe/subscription_pmi_expanded_incomplete_cvc_fail.json:47:            "downloadURL": "https://vpn.mozilla.org/vpn/download?utm_medium=email&utm_source=email&utm_campaign=subscription-download",
packages/fxa-auth-server/test/local/payments/fixtures/stripe/subscription_pmi_expanded_incomplete_cvc_fail.json:67:            "downloadURL": "https://vpn.mozilla.org/vpn/download?utm_medium=email&utm_source=email&utm_campaign=subscription-download",
packages/fxa-auth-server/test/local/payments/fixtures/stripe/subscription_pmi_expanded_incomplete_cvc_fail.json:170:              "downloadURL": "https://vpn.mozilla.org/vpn/download?utm_medium=email&utm_source=email&utm_campaign=subscription-download",
packages/fxa-auth-server/test/local/payments/fixtures/stripe/subscription_pmi_expanded_incomplete_cvc_fail.json:190:              "downloadURL": "https://vpn.mozilla.org/vpn/download?utm_medium=email&utm_source=email&utm_campaign=subscription-download",
packages/fxa-auth-server/test/local/payments/fixtures/stripe/subscription_pmi_expanded_incomplete_cvc_fail.json:481:      "downloadURL": "https://vpn.mozilla.org/vpn/download?utm_medium=email&utm_source=email&utm_campaign=subscription-download",

packages/fxa-settings/src/components/BentoMenu/index.tsx:71:                      href="https://monitor.firefox.com"
packages/fxa-settings/src/components/BentoMenu/index.tsx:125:                      href="https://vpn.mozilla.org/?utm_source=accounts.firefox.com&utm_medium=referral&utm_campaign=fxa-settings&utm_content=bento-promo"

packages/fxa-settings/src/components/ConnectedServices/Service.tsx:50:      serviceLink = 'https://monitor.firefox.com/';

packages/fxa-settings/src/components/ConnectedServices/index.test.tsx:141:          'https://monitor.firefox.com/'
```